CM7/Core/Src/main.c

```diff
@@ -350,11 +351,11 @@ static void MX_ADC1_Init(void)
   hadc1.Instance = ADC1;
   hadc1.Init.ClockPrescaler = ADC_CLOCK_ASYNC_DIV1;
   hadc1.Init.Resolution = ADC_RESOLUTION_16B;
-  hadc1.Init.ScanConvMode = ADC_SCAN_ENABLE;
+  hadc1.Init.ScanConvMode = ADC_SCAN_DISABLE;

```

Comment:
Why are we disabling the Scan Conv mode? I thought this was needed for automatically restarting the ADC, but I def could be wrong

Reply:
Oh, I think that's just a mistake on my end, I'll change that back lol